.github/workflows/test-example-notebooks.yaml

```diff
@@ -0,0 +1,40 @@
+name: Test example notebooks
+
+on:
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  create-pytorchjob-notebook-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-version: ["v1.28.7"]
```

Comment:
@kubeflow/wg-training-leads @Electronic-Waste @saileshd1402 Do we want to run our Notebooks on all supported Kubernetes version similar to E2E tests: https://github.com/kubeflow/training-operator/blob/0ee9ca5b1f5de888f99b18607bee62a9303b6a04/.github/workflows/integration-tests.yaml#L28 ?

Reply:
Yeah, I agree with you. We should expand the tests to all supported Kubernetes versions.